apps/kyc-server/src/components/dashboard/table/user-details-sheet.tsx

```
@@ -0,0 +1,225 @@
+						</SheetHeader>
+
+						<div className="flex-1 space-y-6 overflow-auto">
+							{/* User Information */}
+							<div className="space-y-4">
+								<h3 className="text-lg font-medium">User Information</h3>
+
+								<div className="grid grid-cols-1 gap-4">
+									<div className="space-y-2">
+										<label className="text-sm font-medium text-muted-foreground">
+											User ID
+										</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)